masterlist.yaml

```
@@ -6935,6 +6935,33 @@ plugins:
       - crc: 0xA08B7C20
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'An Unexpected Company( - Better Cities)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/41801/'
+        name: 'An Unexpected Company'
+  - name: 'An Unexpected Company.esp'
+    req: [ *SilentVoice ]
+    inc:
+      - 'A Family Problem.esp'
+      - 'LGVampire.esp'
+    msg:
+      - <<: *obsolete
+        subs: [ '[An Unexpected Company v1-2-2 PATCH (with increased follow distance)](https://www.nexusmods.com/oblivion/mods/41801/)' ]
+        condition: 'checksum("An Unexpected Company.esp", 9E6F5833)'
```

The plugin's version number can be used.

```suggestion
        condition: 'version("An Unexpected Company.esp", "1.2.2", <)'
```